doc/rados/configuration/osd-config-ref.rst

```diff
@@ -438,6 +438,23 @@ Operations
 :Valid Range: 1-63
 
 
+``osd kick recovery op priority``
+
+:Description: The priority set for recovery operations that are forced by
+              client operations.
+              The new "mclock_opclass/mclock_client" queue prioritizes
```

Comment:
I guess in this section you mean to say that this new priority will still be of significance when we switch to the `mclock_opclass` queue?

Reply:
yeah, that's right.